controllers/apiController.js

```diff
@@ -2,24 +2,111 @@ const fs = require('fs');
 const express = require('express');
 const router = express.Router();
 const bodyParser = require('body-parser')
+const MongoClient = require('mongodb').MongoClient;
```

Comment:
you can use ES6 Destructuring to achieve the same http://es6-features.org/#ObjectMatchingShorthandNotation
```js
const { MongoClient, ObjectID } = require('mongodb');
```